src/charm.py

```diff
@@ -1177,5 +1103,8 @@ def _set_data_dir_permissions(container: Container) -> None:
     # END: static methods
 
 
+# END: Secret cache
+
+
```

Comment:
should this be here? It looks like there are other non-secret functions before this. We also have no `START` section. 

suggestions you can either:
1. move it to the correct place and add a `START` section
2. remove it

Reply:
Bah, seems like a leftover as the `mongodb_secrets.py` (new lib) conents were here originally, thank U for noticing!!!